unique_identifier_msgs::msg::UUID object_id
float64 likelihood
---
bool result
